presto-tpch/src/main/java/io/prestosql/plugin/tpch/ColumnNaming.java

```diff
@@ -16,22 +16,22 @@
 import io.prestosql.tpch.TpchColumn;
 import io.prestosql.tpch.TpchEntity;
 
-import java.util.function.Function;
-
 public enum ColumnNaming
 {
-    STANDARD(TpchColumn::getColumnName),
-    SIMPLIFIED(TpchColumn::getSimplifiedColumnName);
-
-    private final Function<TpchColumn<?>, String> columnNameGetter;
-
-    ColumnNaming(Function<TpchColumn<?>, String> columnNameGetter)
-    {
-        this.columnNameGetter = columnNameGetter;
-    }
+    STANDARD {
+        @Override
+        public String getName(TpchColumn<? extends TpchEntity> tpchColumn)
+        {
+            return tpchColumn.getColumnName();
+        }
+    },
+    SIMPLIFIED {
+        @Override
+        public String getName(TpchColumn<? extends TpchEntity> tpchColumn)
+        {
+            return tpchColumn.getSimplifiedColumnName();
+        }
+    };
 
-    public String getName(TpchColumn<? extends TpchEntity> tpchColumn)
-    {
-        return columnNameGetter.apply(tpchColumn);
-    }
+    public abstract String getName(TpchColumn<? extends TpchEntity> tpchColumn);
```

Comment:
see [ImmutableEnumChecker](https://errorprone.info/bugpattern/ImmutableEnumChecker)

```
TIP: Instead of creating an enum with functional interface fields (Predicate, Function, etc.), declare abstract methods that are overridden by each constant. For example, do this:
```